.github/workflows/review.yml

```diff
@@ -16,15 +16,19 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-node@v4
       with:
-        node-version: 20
-    - name: Review PR
-      env:
-        GH_TOKEN: ${{ github.token }}
+        node-version: 22
+    - name: Verify Review Script
       run: |
         git fetch https://github.com/safe-global/safe-deployments.git main:safe-global-main
-        if ! git diff --exit-code safe-global-main -- bin/github-review.sh; then
-          echo "ERROR: GitHub review script is not up-to-date" 1>&2
+        if ! git diff --exit-code safe-global-main -- . ':!src/assets'; then
+          echo "ERROR: GitHub review script may not be up-to-date" 1>&2
```

Comment:
I would still keep this as before, as this particular task is to check the review script changes.

We already have a check for changes other than assets here: https://github.com/safe-global/safe-deployments/blob/bed58cc0f620c8fa08d1fd83483fc2e3a6b8cb7e/bin/github-review.sh#L67

Reply:
You are correct - the reason I changed it was that there is a hidden dependency on the `git diff` check you mentioned being in the `github-review.sh` script. If we refactor it into a `.js` script, we would accidentally break the protection. Also the command remains simple enough IMO that the change doesn't make the CI workflow more complicated.